rero_mef/api_mef.py

```diff
@@ -106,58 +96,79 @@ def get_all_pids_without_viaf(cls):
             yield hit.pid
 
     @classmethod
-    def get_pids_with_multiple_mef(cls, record_types=None, verbose=False):
+    def get_pids_with_multiple_mef(cls, record_types=None, verbose=False,
+                                   before=1):
         """Get agent pids with multiple MEF records.
 
         :params record_types: Record types (pid_types).
         :param verbose: Verbose.
+        :param before: Get multiple MEF before x minutes (default 1 minute).
         :returns: pids, multiple pids, missing pids.
         """
         pids = {}
         multiple_pids = {}
         missing_pids = {}
-        if record_types:
-            for record_type in record_types:
-                if verbose:
-                    click.echo(f'Calculating {record_type}:')
+        none_pids = {}
+        agents = {}
+        if not record_types:
+            record_types = []
```

Comment:
`record_types = record_types or []` --> single line. Maybe moves this line at top of the function